script.js

```diff
@@ -1,4 +1,215 @@
+}
+
+var hitOrStand = function(input){
+  if(input == ''){
+    var playerCard = deck.pop()
+    playerTotal += playerCard.rank
+    if(playerCard.rank == 1){
+      GAMEMODE = 'ace'
+      playerTotal -= playerCard.rank
+      return `you got ${playerCard.name} of ${playerCard.suit}<br> your current total is ${playerTotal}, input '1' or '11'`
+    }
+    if(playerTotal >21){
+    GAMEMODE = 'bet'
+    playerPoint -= betNumber
+    dealerPoint += betNumber
+    return `you got ${playerCard.name} of ${playerCard.suit}<br>your total is ${playerTotal}<br>bustedd!<br><br>${username} point :${playerPoint}<br>dealer point :${dealerPoint}<br><br>enter number of point you want to bet`
+    }
+    else{
+      return `you got ${playerCard.name} of ${playerCard.suit}<br>your total is ${playerTotal}<br>click submit to add the card or input "stand" if you are done`
+    }
+  }
+    GAMEMODE = 'win or lose'
+    return `your total is ${playerTotal}, click submit to decide the winner`
+}
+
+var aceChose = function(input){
+  if(input == '1'){
+    playerTotal +=1
+    if(playerTotal >21){
+    GAMEMODE = 'bet'
+    return `you chose 1<br>your total is ${playerTotal}<br>bustedd!<br><br>${username} point :${playerPoint}<br>dealer point :${dealerPoint}<br><br>enter number of point you want to bet`
+    }
+    GAMEMODE = 'hit or stand'
+    return `you chose 1<br>your total is ${playerTotal}<br>click submit to add the card or input "stand" if you are done`
+  }
+  else if(input == '11'){
+    playerTotal +=11
+    if(playerTotal >21){
+    GAMEMODE = `bet`
+    return `you chose 11<br>your total is ${playerTotal}<br>bustedd!<br><br>${username} point :${playerPoint}<br>dealer point :${dealerPoint}<br><br>enter number of point you want to bet`
+    }
+    GAMEMODE = 'hit or stand'
+    return `you chose 11<br>your total is ${playerTotal}<br>click submit to add the card or input "stand" if you are done`
+  }
+  else{
+    return `your current total is ${playerTotal}, you got ace input '1' or '11'`
+  }
+
+}
+
+var winOrLose = function(){
+  while(dealerTotal < 17){
+    dealerCard = deck.pop()
+    dealerTotal += dealerCard.rank
+  }
+  // player win dealer busted
+  if(dealerTotal >21){
+    GAMEMODE = 'bet'
+    playerPoint += betNumber
+    dealerPoint -= betNumber
+    return `Dealer got ${dealerTotal}<br>bustedd! YOU WINN<br><br>${username} point :${playerPoint}<br>dealer point :${dealerPoint}<br><br>enter number of point you want to bet`
+    }
+   GAMEMODE = 'bet'
+   // player total > dealer total
+  if(dealerTotal > playerTotal){
+    playerPoint -= betNumber
+    dealerPoint += betNumber
+    return `${username} sum : ${playerTotal} <br>dealer sum : ${dealerTotal}<br><br>dealer winn<br><br>${username} point :${playerPoint}<br>dealer point :${dealerPoint}<br><br>enter number of point you want to bet`
+  }
+  // player total < dealer total
```

Comment:
good attempt at commenting. Perhaps the comments could be more descriptive since right now the comments are the same as the condition of the if statement.